authentication/src/main/java/io/camunda/authentication/entity/CamundaUser.java

```diff
@@ -105,13 +108,20 @@ public String getSalesPlanType() {
   }
 
   public Map<ClusterMetadata.AppName, String> getC8Links() {
+    if (c8Links == null) {
+      return Collections.emptyMap();
```

Comment:
Can we do this in the constructor or is there a case where we need to distinguish null and the empty map?

Reply:
Yes, I'm sorry, that could have been clearer. It's unfortunate that we normalize in multiple places now, but if that is too annoying we could always make the constructors call `this()`.